packages/www/pages/contact.tsx

```diff
@@ -150,187 +133,16 @@ const ContactPage = () => {
-                      mb: "$2",
-                    }}>
-                    <StyledCircleRadio
-                      value="PRODUCT_ERROR"
-                      id="PRODUCT_ERROR"
-                    />
-                    <Box
-                      as="label"
-                      htmlFor="PRODUCT_ERROR"
-                      css={{ color: "$slate11", fontSize: "$2" }}>
-                      Product error
-                    </Box>
-                  </Box>
-                  <Box
-                    css={{
-                      display: "flex",
-                      alignItems: "center",
-                      justifyContent: "flex-start",
-                      mb: "$2",
-                    }}>
-                    <StyledCircleRadio
-                      value="BILLING_ISSUE"
-                      id="BILLING_ISSUE"
-                    />
-                    <Box
-                      as="label"
-                      htmlFor="BILLING_ISSUE"
-                      css={{ color: "$slate11", fontSize: "$2" }}>
-                      Billing issue
-                    </Box>
-                  </Box>
-                  <Box
-                    css={{
-                      display: "flex",
-                      alignItems: "center",
-                      justifyContent: "flex-start",
-                      mb: "$2",
-                    }}>
-                    <StyledCircleRadio
-                      value="MistServer"
-                      id="FEATURE_REQUEST"
-                    />
-                    <Box
-                      as="label"
-                      htmlFor="FEATURE_REQUEST"
-                      css={{ color: "$slate11", fontSize: "$2" }}>
-                      Feature request
-                    </Box>
-                  </Box>
-                  <Box
-                    css={{
-                      display: "flex",
-                      alignItems: "center",
-                      justifyContent: "flex-start",
-                      mb: "$2",
-                    }}>
-                    <StyledCircleRadio
-                      value="GENERAL_INQUIRY"
-                      id="GENERAL_INQUIRY"
-                    />
-                    <Box
-                      as="label"
-                      htmlFor="GENERAL_INQUIRY"
-                      css={{ color: "$slate11", fontSize: "$2" }}>
-                      General inquiry
-                    </Box>
-                  </Box>
-                </RadioGroup>
-              </Box>
-            </Grid>
 
             <TextArea
               size="3"
               id="message"
               css={{ width: "100%", boxSizing: "border-box", mb: "$4" }}
               name="TICKET.content"
-              placeholder="Tell us what's happening*"
+              placeholder="Please include contexts like links, error messages, or logs based on the type of question you have. The more information, the better.*"
```

Comment:
What do you think about making this field more general for folks that might not have a question related to support?
```
How can we help you? To guide the discussion, please give us specific details on
your interest in speaking with our team. The more information, the better.
```

Reply:
That makes sense